.github/workflows/check.yml

```diff
@@ -11,17 +11,17 @@ jobs:
         include:
           - elixir: '1.10.4'
             otp: '23.0.2'
-          - elixir: '1.7.0'
-            otp: '19.0'
+          - elixir: '1.7.4'
+            otp: '19.3.6'
 
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
I wouldn't want test suite to start failing because GitHub switches their latest Ubuntu. This already happened to me once (not sure if in this project). Latest Ubuntu 20 vastly differs in which build deps are provided vs which need to be installed and how. If we're to support Ubuntu 20 then I'm more for adding matrix and checking both (since 18 is still very popular) or switching explicitly.